src/models/pmv_ppd.js

```diff
@@ -0,0 +1,486 @@
+import {
+  check_standard_compliance_array,
+  round,
+  units_converter,
+  units_converter_array,
+  valid_range,
+} from "../utilities/utilities.js";
+import { cooling_effect } from "./cooling_effect.js";
+
+/**
+ * @typedef {Object} Pmv_ppdKwargs
+ * @property {'SI'|'IP'} units - select the SI (International System of Units) or the IP (Imperial Units) system.
+ * @property { boolean } limit_inputs - Default is True. By default, if the inputs are outside the standard applicability
+ *    limits the function returns NaN. If false, returns pmv and ppd values even if input values are outside
+ *    the applicability limits of the model.
+ *    The ASHRAE 55 2020 limits are 10 < tdb [°C] < 40, 10 < tr [°C] < 40,
+ *    0 < vr [m/s] < 2, 1 < met [met] < 4, and 0 < clo [clo] < 1.5.
+ *    The ISO 7730 2005 limits are 10 < tdb [°C] < 30, 10 < tr [°C] < 40,
+ *    0 < vr [m/s] < 1, 0.8 < met [met] < 4, 0 < clo [clo] < 2, and -2 < PMV < 2.
+ * @property { boolean } airspeed_control - This only applies if standard = "ASHRAE".
+ *    Default is True. By default, it is assumed that the occupant has control over the airspeed.
+ *    In this case, the ASHRAE 55 Standard does not impose any airspeed limits.
+ *    On the other hand, if the occupant has no control over the airspeed,
+ *    the ASHRAE 55 imposes an upper limit for v which varies as a function of
+ *    the operative temperature, for more information please consult the Standard.
+ * @public
+ */
+
+/**
+ * Returns Predicted Mean Vote (`PMV`) and Predicted Percentage of
+ * Dissatisfied (`PPD`) calculated in accordance with main thermal comfort
+ * Standards. The PMV is an index that predicts the mean value of the thermal
+ * sensation votes (self-reported perceptions) of a large group of people on a
+ * sensation scale expressed from –3 to +3 corresponding to the categories:
+ * cold, cool, slightly cool, neutral, slightly warm, warm, and hot. {@link #ref_1|[1]}
+ *
+ * While the PMV equation is the same for both the ISO and ASHRAE standards, in the
+ * ASHRAE 55 PMV equation, the SET is used to calculate the cooling effect first,
+ * this is then subtracted from both the air and mean radiant temperatures, and the
+ * differences are used as input to the PMV model, while the airspeed is set to 0.1m/s.
+ * Please read more in the Note below.
+ *
+ * Notes:
+ * You can use this function to calculate the `PMV` and `PPD` in accordance with
+ * either the ASHRAE 55 2020 Standard {@link #ref_1|[1]} or the ISO 7730 Standard {@link #ref_2|[2]}.
+ *
+ * _PMV:
+ * {@link https://en.wikipedia.org/wiki/Thermal_comfort#PMV/PPD_method|PMV}
```

Comment:
Maybe we can put these links in the paragraph? so instead of `PMV` you could do what you are doing here

Reply:
done